docs/style.md

```diff
@@ -27,9 +27,9 @@ instead of
 use crate::Something;
 
 use tokio::task::JoinHandle;
-use supermusr_common::Time
-
 use std::time::Duration
+
+use supermusr_common::Time
```

Comment:
I only meant you to sort the example above, this snippet is supposed to be an example of what not to do.